chapters/lexicalstructure.tex

```diff
@@ -83,7 +83,8 @@ \subsection{Identifiers}\doublelabel{identifiers}
 escapes.
 The single quotes are part of the identifier, i.e., \lstinline!'x'! and \lstinline!x!
 are distinct identifiers, but the redundant escapes (\lstinline!'\?'! and \lstinline!'\"'!) are the same as the corresponding non-escaped
-variants (\lstinline!'?'! and \lstinline!'"'!). A full BNF definition of the Modelica syntax and
+variants (\lstinline!'?'! and \lstinline!'"'!), and outside of Modelica source code the redundant escapes shall not be used.
```

Comment:
Might be easier to grasp if that long sentence is split in two or three parts.

Reply:
Agreed, and I tried to do that.